Changed boot screens and added Speed2

Freezer-SD opened the pull request:
More clear boot sequence and added ESP-GPS logo

RP6conrad commented:
The bitmap font file Fonts/FreeSansBold120pt7b.h (17084) is very large, as all characters are included. There is tool which permits to have only the used digits in the file, which will make this file a lot smaller. Take a look :https://tchapi.github.io/Adafruit-GFX-Font-Customiser/ used for extracting only needed digits and decimal point !!
Greetings, Jan.
Verzonden vanuit Mail voor Windows

Van: Freezer-SD
Verzonden: zondag 30 april 2023 18:17
Aan: RP6conrad/ESP-GPS-Logger
CC: Subscribed
Onderwerp: [RP6conrad/ESP-GPS-Logger] Changed boot screens and added Speed2 (PR#21)

More clear boot sequence and added ESP-GPS logo

You can view, comment on, or merge this pull request online at:
  https://github.com/RP6conrad/ESP-GPS-Logger/pull/21
Commit Summary
• 986bb56 Changed boot screens and added Speed2
File Changes 
(8 files)
• M E_paper.cpp (1198) 
• M E_paper.h (9) 
• M Fonts/BitmapSurfbuddies.h (22) 
• A Fonts/FreeSansBold120pt7b.h (17084) 
• A Fonts/FreeSansBold30pt7b.h (1172) 
• A Fonts/FreeSansBold60pt7b.h (4370) 
• A Fonts/FreeSansBold75pt7b.h (6735) 
• M Rtos5.ino (8) 
Patch Links:
• https://github.com/RP6conrad/ESP-GPS-Logger/pull/21.patch
• https://github.com/RP6conrad/ESP-GPS-Logger/pull/21.diff
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Freezer-SD commented:
The 120pt can be deleted since it is not used. The Speed2 screen uses a 75pt for the integer and 30pt behind the dot.

I’ve reduced the additional fonts. And deleted the 120pt.

From: RP6conrad ***@***.***>
Sent: zondag 30 april 2023 19:22
To: RP6conrad/ESP-GPS-Logger ***@***.***>
Cc: Freezer-SD ***@***.***>; Author ***@***.***>
Subject: Re: [RP6conrad/ESP-GPS-Logger] Changed boot screens and added Speed2 (PR #21)

The bitmap font file Fonts/FreeSansBold120pt7b.h (17084) is very large, as all characters are included. There is tool which permits to have only the used digits in the file, which will make this file a lot smaller. Take a look :https://tchapi.github.io/Adafruit-GFX-Font-Customiser/ used for extracting only needed digits and decimal point !!
Greetings, Jan.
Verzonden vanuit Mail voor Windows

Van: Freezer-SD
Verzonden: zondag 30 april 2023 18:17
Aan: RP6conrad/ESP-GPS-Logger
CC: Subscribed
Onderwerp: [RP6conrad/ESP-GPS-Logger] Changed boot screens and added Speed2 (PR#21)

More clear boot sequence and added ESP-GPS logo

You can view, comment on, or merge this pull request online at:
  https://github.com/RP6conrad/ESP-GPS-Logger/pull/21
Commit Summary
• 986bb56 Changed boot screens and added Speed2
File Changes
(8 files)
• M E_paper.cpp (1198)
• M E_paper.h (9)
• M Fonts/BitmapSurfbuddies.h (22)
• A Fonts/FreeSansBold120pt7b.h (17084)
• A Fonts/FreeSansBold30pt7b.h (1172)
• A Fonts/FreeSansBold60pt7b.h (4370)
• A Fonts/FreeSansBold75pt7b.h (6735)
• M Rtos5.ino (8)
Patch Links:
• https://github.com/RP6conrad/ESP-GPS-Logger/pull/21.patch
• https://github.com/RP6conrad/ESP-GPS-Logger/pull/21.diff
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***<mailto:***@***.***>>



—
Reply to this email directly, view it on GitHub<https://github.com/RP6conrad/ESP-GPS-Logger/pull/21#issuecomment-1529085871>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXUAXP7KLD4RXWAVESDY7R3XD2NUFANCNFSM6AAAAAAXRAPOEM>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
